.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release
```

[suggest] Considering consistency with other workflows, `...ing` seems better:

```suggestion
name: Releasing
```

And `release.yml` → `releasing.yml`.

---

Sure.